app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -92,17 +93,35 @@ class HomeMenu(
         }
     }
 
+    val syncedTabsItem = BrowserMenuImageText(
+        context.getString(R.string.synced_tabs),
+        R.drawable.ic_synced_tabs,
+        primaryTextColor
+    ) {
+        onItemTapped.invoke(Item.SyncedTabs)
+    }
+
+    private var signedInToFxa = false
```

Comment:
nit: can be removed as well since it's unused.

Reply:
It's not unused. See lines 110 and 122

Reply:
Ah, my mistake! 🙂